pkg/kubelet/prober/prober.go

```diff
@@ -112,6 +112,9 @@ func (pb *prober) probe(ctx context.Context, probeType probeType, pod *v1.Pod, s
 	if result == probe.Warning {
 		pb.recordContainerEvent(pod, &container, v1.EventTypeWarning, events.ContainerProbeWarning, "%s probe warning: %s", probeType, output)
 		klog.V(3).InfoS("Probe succeeded with a warning", "probeType", probeType, "pod", klog.KObj(pod), "podUID", pod.UID, "containerName", container.Name, "output", output)
+	} else if result == probe.Unknown {
+		klog.V(3).InfoS("Probe result in unknown state", "probeType", probeType, "pod", klog.KObj(pod), "podUID", pod.UID, "containerName", container.Name)
```

Comment:
Would it make sense to somehow state that the probe actually succeded, similar to what's done for probe.Warning above?

Reply:
it did made sense. I updated it